### PR TITLE
VCF header modified

### PR DESCRIPTION
**Pull request by nuin**
I made the header with actual fields that appear in the INFO column

Also some small cosmetic changes and commented out some imports that were unused in some of the files (can be removed if that's the case).

There were some wrong variable/structure names in the dar generator.

I will keep working on this branch for the tab delimited generation.

### CODE REVIEW

File: src/generators/daf_file_generator.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
-import sys
+# import sys
```
Comment (a8wright) [inline src/generators/daf_file_generator.py]: Can you delete all the parts that you are commenting out if they are not required?

File: src/generators/daf_file_generator.py [hunk 2]
```diff
@@ -81,15 +81,15 @@ def generate_file(self):
...
-            genetic_sex = ""
+            # gene_product_form_id = ""
+            # additional_genetic_component = ""
+            # experimental_conditions = ""
+            # qualifier = ""
+            # modifier_association_type = ""
+            # modifier_qualifier = ""
+            # modifier_genetic = ""
+            # modifier_experimental_conditions = ""
+            # genetic_sex = ""
```
Comment (a8wright) [inline src/generators/daf_file_generator.py]: I am not sure if these should be deleted. We need to get the latest spec to see if they should be added or not. I am leaning towards deleting them.

File: src/generators/vcf_file_generator.py [hunk 3]
```diff
@@ -20,20 +20,14 @@ class VcfFileGenerator:
...
-##INFO=<ID=symbol,Type=String,Number=0,Description="The human readable name of the allele">
+##INFO=<ID=alleles,Type=String,Description="The alleles of the variant">
+##INFO=<ID=hgvs_nomenclature,Type=String,Description="the HGVS name of the allele">
+##INFO=<ID=symbol,Type=String,Description="The human readable name of the allele">
+##INFO=<ID=geneLevelConsequence,Type=String,Description="VEP consequence of the variant">
+##INFO=<ID=symbol_text,Type=String,Description="Another human readable representation of the allele">
 ##phasing=partial
-##reference=
-##source=agr_file_generator"""
+##source=AGR VCF File generator"""
```
Comment (a8wright) [inline src/generators/vcf_file_generator.py]: I am not sure if you are or not but you should make sure that the header is sorted according to the command line sort command.

File: src/generators/daf_file_generator.py (hunk 2 above)
Comment (nuin) [reply src/generators/daf_file_generator.py]: I just commented out as they are not used ATM, but might be useful in the future.

File: src/generators/vcf_file_generator.py (hunk 3 above)
Comment (nuin) [reply src/generators/vcf_file_generator.py]: I will work on that.

**Pull request closed without merging**